packages/dcos-signal/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-signal.git",
-    "ref": "7e7cb7a62cb27c175e1864eca98fde173d8267b7",
-    "ref_origin": "1.1.9"
+    "ref": "008fb3631cc90449e80cbb5f404428af1efe504d",
+    "ref_origin": "karlkfi/gen_platform"
```

Comment:
@karlkfi I incremented the release to tag `1.3.0`, please point this `ref` to `40a23b4dcba6b3053379235a96bc3ac53996e146` and `ref_origin` `1.3.0`